docs/modules/develop/pages/classes/models.adoc

```diff
@@ -0,0 +1,90 @@
+= Models
+
+The models in Decidim are just plain `ActiveRecord` models, so you can use all the methods that come with it.
+
+A model class example would be:
+
+```ruby
+module Decidim
+  module MyPlugin
+    class MyResource < Decidim::MyPlugin::ApplicationRecord
+      include Decidim::HasComponent
+      include Decidim::Traceable
+      include Decidim::Loggable
+      include Decidim::Authorable
+      include Decidim::TranslatableResource
+    end
+  end
+end
+```
+
+
+== Scopes
+
+Within core provided classes there are no `default_scopes`.
+
+== Validations
+
+All the validations are defined in the models themselves.
+
+== Callbacks
+
+The number of the callbacks is reduced to the minimum.
+
+== Concerns
+
+The concerns are used to share code between models. They are located in `app/models/concerns`.
+
+Most commonly used concerns are:
```

Comment:
Some of these have pages in docs, can you add the links 🙏🏽 ? 

![Screenshot of the TOC of docs](https://github.com/decidim/decidim/assets/717367/3d6cc412-a4a4-40c5-a625-83daf610ebc3)